lib/src/widgets/chat/ds_contact_message_bubble.widget.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSContactMessageBubble extends StatelessWidget {
+  final String? name;
+  final String? phone;
+  final String? email;
+  final String? address;
+  final DSAlign align;
+  final List<DSBorderRadius> borderRadius;
+  final DSMessageBubbleStyle style;
+
+  DSContactMessageBubble({
+    super.key,
+    required this.name,
+    required this.phone,
+    required this.address,
+    required this.email,
+    required this.align,
+    this.borderRadius = const [DSBorderRadius.all],
+    DSMessageBubbleStyle? style,
+  }) : style = style ?? DSMessageBubbleStyle();
+
+  @override
+  Widget build(BuildContext context) {
+    return DSMessageBubble(
+      align: align,
+      borderRadius: borderRadius,
+      padding: const EdgeInsets.all(16.0),
+      style: style,
+      child: _buildContactCard(),
+    );
+  }
+
+  Widget _buildContactCard() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Visibility(
+          visible: name != null,
+          child: DSBodyText(
+            name,
+            fontWeight: DSFontWeights.semiBold,
+          ),
+        ),
+        const SizedBox(height: 16.0),
+        Visibility(
+          visible: phone != null,
+          child: Padding(
+            padding: const EdgeInsets.only(bottom: 4.0),
+            child: _buildContactField(
+              title: 'Telefone',
+              body: phone!,
+            ),
+          ),
+        ),
+        Visibility(
+          visible: email != null,
+          child: Padding(
+            padding: const EdgeInsets.only(bottom: 4.0),
+            child: _buildContactField(
+              title: 'E-mail',
+              body: email!,
+            ),
+          ),
+        ),
+        Visibility(
```

Comment:
You should avoid the use of ```Null assertion operators``` in a ```Visibility``` child, because it will throw exception when building the widget.

<img width="321" alt="image" src="https://github.com/takenet/blip-ds-flutter/assets/91232521/645e17b5-146f-4ef2-955a-98972e6f1c43">

Suggestion:
```dart
if(address != null)
          _buildContactField(
            title: 'Endereço',
            body: address!,
        ),
```

Ref: https://dart.dev/null-safety/understanding-null-safety#null-assertion-operator